app/src/main/res/xml/preferences.xml

```diff
@@ -104,6 +104,14 @@
                 android:title="@string/pref_name_ui_article_textAlignment_justify"
                 android:summary="@string/pref_desc_ui_article_textAlignment_justify"
                 android:defaultValue="true"/>
+            <ListPreference
+                android:key="@string/pref_key_ui_article_handlePreformattedText"
+                android:title="@string/pref_name_ui_article_handlePreformattedText"
+                android:summary="@string/pref_desc_ui_article_handlePreformattedText"
+                android:dialogTitle="@string/pref_name_ui_article_handlePreformattedText"
+                android:entries="@array/pref_option_handlePreformattedText_values"
+                android:entryValues="@array/pref_option_handlePreformattedText_values"
```

Comment:
`android:entries` is for user-facing localized values and `android:entryValues` is for internal values. See `pref_option_autoSync` for example.

Reply:
Should be changed now. I used the entries for the CSS-classes, I hope this is correct.